README.md

```diff
@@ -39,6 +39,22 @@ the Kubernetes world:
 [5]: https://github.com/openshift/machine-config-operator/tree/master/docs
 [6]: https://github.com/UKHomeOffice/seccomp-config
 
+
+## Features
+
+The SPO's features are implemented for each one of the underlying
+supported technologies, namely: Seccomp, SELinux and AppArmor. 
+Here's the main features' status:
+
+|                                  | Seccomp | SELinux | AppArmor |
+|----------------------------------|---------|---------|----------|
+|             Custom Resource Type |   Yes   |   Yes   |    Yes   |
+|       Deploy profiles into nodes |   Yes   |   Yes   |    WIP   |
+| Remove profiles no longer in use |   Yes   |   Yes   |    WIP   |
+|   Profile Auto-generation (logs) |   Yes   |   WIP   |    No    |
+|   Profile Auto-generation (ebpf) |   WIP   |   No    |    No    |
+|             Audit log enrichment |   Yes   |   WIP   |    Yes   |
```

Comment:
What about ProfileBindings? I think that's a pretty neat feature which we plan to add SELinux support for as well.

Reply:
Agreed - completely forgot about that one. :sweat_smile: Added to the list.